.github/workflows/kubespray.yml

```
@@ -0,0 +1,290 @@
+        path: ./logs
+    - name: Upload events
+      if: always()
+      uses: actions/upload-artifact@v2
+      with:
+        name: compliantkubernetes-apps-events
+        path: ./events
+
+    - name: Destroy terraform
+      if: always()
+      uses: docker://hashicorp/terraform:0.14.7
+      with:
+        args: destroy -auto-approve -var-file apps/pipeline/config/exoscale/pipeline-${{ matrix.cluster }}-config/default.tfvars -state apps/pipeline/config/exoscale/pipeline-${{ matrix.cluster }}-config/terraform.tfstate kubespray/contrib/terraform/exoscale
+
+    - name: Prepare DNS cleanup
+      if: always() && (matrix.cluster == 'sc')
+      run: |
+        sed -i "s/CREATE/DELETE/g" apps/pipeline/config/exoscale/dns-${{ matrix.cluster }}.json
+        cat apps/pipeline/config/exoscale/dns-${{ matrix.cluster }}.json
+    - name: Delete DNS records
+      if: always() && (matrix.cluster == 'sc')
+      uses: docker://amazon/aws-cli:2.1.26
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_DNS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_DNS_SECRET_ACCESS_KEY }}
+        AWS_EC2_METADATA_DISABLED: true
+      with:
+        args: route53 change-resource-record-sets --hosted-zone-id Z2STJRQSJO5PZ0 --change-batch file://apps/pipeline/config/exoscale/dns-${{ matrix.cluster }}.json
+
+  buckets-cleanup:
+    needs: apps
+    if: always()
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout bucket-manager
+      uses: actions/checkout@v2
+      with:
+        repository: elastisys/bucket-manager
+        ref: main
+        token: ${{ secrets.CK8S_PIPELINE_GITHUB_TOKEN }}
+    - name: Delete buckets
+      env:
+        EXOSCALE_API_KEY: ${{ secrets.EXOSCALE_API_KEY }}
+        EXOSCALE_API_SECRET: ${{ secrets.EXOSCALE_API_SECRET }}
+      run: |
+        python3 -m venv venv
+        source venv/bin/activate
+        pip install -r S3/requirements.txt
+        ./S3/gen-s3cfg.sh exoscale "${EXOSCALE_API_KEY}" "${EXOSCALE_API_SECRET}" sos-ch-gva-2.exo.io > s3cfg.ini
+        ./S3/manager.sh --s3cfg s3cfg.ini --delete pipeline-exoscale-harbor pipeline-exoscale-velero pipeline-exoscale-es-backup pipeline-exoscale-influxdb pipeline-exoscale-sc-logs
+
+  clean-apps-image:
+    needs: apps
+    if: always()
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout compliantkubernetes-apps
+      uses: actions/checkout@v2
+      with:
+        repository: elastisys/compliantkubernetes-apps
+        ref: main
+    - name: Cleanup docker image
+      run: ./pipeline/cleanup-docker-image.bash
+      env:
+        DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+        DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
+        DOCKERHUB_REPOSITORY: elastisys/compliantkubernetes-apps-pipeline
+
+  alert:
+    runs-on: ubuntu-20.04
+    needs: [buckets-cleanup, clean-apps-image]
+    if: always() && github.ref == 'refs/heads/master'  # Make sure this runs even if some jobs fail, and only on master
```

Since the main branch is named `main` in this repo, this one should check for `main` instead.
```suggestion
    if: always() && github.ref == 'refs/heads/main'  # Make sure this runs even if some jobs fail, and only on main
```

---

Well spotted, thanks!